stellargraph/data/explorer.py

```diff
@@ -51,42 +49,18 @@ def __init__(self, graph, graph_schema=None, seed=None):
 
         # We require a StellarGraph for this
         if not isinstance(graph, StellarGraphBase):
-            raise TypeError(
-                "Graph must be a StellarGraph or StellarDiGraph to use heterogeneous sampling."
-            )
+            raise TypeError("Graph must be a StellarGraph or StellarDiGraph.")
 
         if not graph_schema:
             self.graph_schema = self.graph.create_graph_schema(create_type_maps=True)
         else:
             self.graph_schema = graph_schema
 
-        if self.graph_schema is not None and type(self.graph_schema) is not GraphSchema:
-            raise ValueError(
-                "({}) The parameter graph_schema should be either None or of type GraphSchema.".format(
-                    type(self).__name__
-                )
+        if type(self.graph_schema) is not GraphSchema:
```

Comment:
can this ever break, e.g., if after line 55 self.graph_schema is still None? If yes, this should be reverted to what it was before.

Reply:
Only if _create_graph_schema() can ever return None, which should be an error.